benchmarks/file_resource_benchmarks.cpp

```diff
@@ -107,35 +107,53 @@ void benchmark(std::string name){
-    std::cout << "  Scale Time:          " << std::chrono::duration<double>(end_scale - begin_scale).count() << " sec" <<std::endl;
+    std::cout << "  Scale:               " << ((2 * sizeof(size_t) * iterations) * 1.0E-6)
+              /std::chrono::duration<double>(end_scale - begin_scale).count() << " MB/sec" << std::endl;
+    std::cout << "  Scale Time:          " << std::chrono::duration<double>
+              (end_scale - begin_scale).count() << " sec" <<std::endl;
     std::cout << "  ---------------------------------------\n";
-    std::cout << "  Add:                 " << ((3 * sizeof(size_t) * iterations) * 1.0E-6)/std::chrono::duration<double>(end_add - begin_add).count() << " MB/sec" << std::endl;
-    std::cout << "  Add Time:            " << std::chrono::duration<double>(end_add - begin_add).count() << " sec" <<std::endl;
+    std::cout << "  Add:                 " << ((3 * sizeof(size_t) * iterations) * 1.0E-6)
+              /std::chrono::duration<double>(end_add - begin_add).count() << " MB/sec" << std::endl;
+    std::cout << "  Add Time:            " << std::chrono::duration<double>
+              (end_add - begin_add).count() << " sec" <<std::endl;
     std::cout << "  ---------------------------------------\n";
-    std::cout << "  Triad:               " << ((3 * sizeof(size_t) * iterations) * 1.0E-6)/std::chrono::duration<double>(end_triad - begin_triad).count() << " MB/sec" << std::endl;
-    std::cout << "  Triad Time:          " << std::chrono::duration<double>(end_triad - begin_triad).count() << " sec" <<std::endl;
+    std::cout << "  Triad:               " << ((3 * sizeof(size_t) * iterations) * 1.0E-6)
+              /std::chrono::duration<double>(end_triad - begin_triad).count() << " MB/sec" << std::endl;
+    std::cout << "  Triad Time:          " << std::chrono::duration<double>
+              (end_triad - begin_triad).count() << " sec" <<std::endl;
     std::cout << "  ---------------------------------------\n";
-    std::cout << "  Total Time:          " << std::chrono::duration<double>(end - begin).count() << " sec" <<std::endl<<std::endl;
+    std::cout << "  Total Time:          " << std::chrono::duration<double>(end - begin).count() 
+              << " sec" <<std::endl<<std::endl;
 }
 
 int main(int, char** argv) {
     iterations = atoi(argv[1]);
-    std::cout << "Array Size:   " << iterations << "        Memory Size: " << (double)((sizeof(size_t)*iterations)* 1.0E-6) << " MB" << std::endl;
-    std::cout << "Total Arrays: 3       " << "        Total Memory Size: " << (double)((3*sizeof(size_t)*iterations)* 1.0E-6) << " MB" << std::endl << std::endl;
+    std::cout << "Array Size:   " << iterations << "        Memory Size: " 
+              << (double)((sizeof(size_t)*iterations)* 1.0E-6) << " MB" << std::endl;
+    std::cout << "Total Arrays: 3       " << "        Total Memory Size: " 
+              << (double)((3*sizeof(size_t)*iterations)* 1.0E-6) << " MB" << std::endl << std::endl;
```

Comment:
Were the changes above this comment done automatically by our style checker?  It's difficult to see if there are any substantive changes here.

Reply:
No this was me being OCD - is this ok to keep as is?

Reply:
This is fine.  From the side-by-side diffs, it was difficult to parse out what might have changed.  When we run this through clang, will it change again?

Reply:
TBH, I thought we already did that a while back (like to the then current develop branch) and I'm not sure why this wasn't changed back then. But yea, I'm not sure.